src/api/app/queries/autocomplete_finder_base.rb

```diff
@@ -0,0 +1,7 @@
+class AutocompleteFinderBase
+  def initialize(relation, search_criteria, limit = 50)
```

Comment:
What do you think about `limit` being a keyword argument? It would look like this and make this a bit clearer when the `AutocompleteFinder` classes are initialized.

```suggestion
  def initialize(relation, search_criteria, limit: 50)
```